www/kubl05/SP1/about.html

```diff
@@ -20,8 +22,9 @@
     <div class="wrapper">
         <div class="container-top">
             <header>
-                <a href="index.html"><img class="logo" src="img/logo.png" onmouseover="this.src='img/logo-hover.png';"
-                        onmouseout="this.src='img/logo.png';" alt="Zde má být logo"></a>
+                <a href="index.html">
+                    <div class="logo"></div>
+                </a>
                 <span class="title">Lukáš Kubík</span>
```

Comment:
![image](https://user-images.githubusercontent.com/20724910/47803455-d1a2a280-dd32-11e8-80de-7a20174d357b.png)

mensi bug kdyz zkousim resizovat okno

Reply:
Nešlo o nějakou ad-hoc věc? Mně se ničeho podobného nedaří docílit, a to resizuju jako zběsilý :(

Reply:
muzes tomu spanu dat pevnou width timse to vyresi